create_env.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+pyv=$(python3 -c 'import sys; print(sys.version_info[1])')
+echo $pyv
+
+if [[ "$pyv" -eq "9" ]]
+then 
+    echo "Valid version"
+    python3 -m venv --clear venv_quant_course_v39
+    source ./venv_quant_course_v39/bin/activate #lehet a Scriptben van bizonyos distrokon
+    python3 -m pip install --upgrade pip
+    pip install -r QuantCourseBP/requirements.txt #ha más útvonalon van a txt azt adjuk meg
```

Comment:
I think the location should be the following here:
```suggestion
    pip install -r requirements.txt
```